app/add_note.py

```diff
@@ -42,12 +42,14 @@ def get(self):
             return self.error(404, _('No person id was specified.'))
         try:
             person = Person.get(self.repo, self.params.id)
+        # TODO(ichikawa) Consider removing this "except" clause.
```

Comment:
I also checked the code and I don't think it will raise ValueError. Why not just remove it?